flexget/manager.py

```diff
@@ -921,20 +922,20 @@ def acquire_lock(self, event: bool = True) -> Iterator:
     def write_lock(self, ipc_info: dict = None) -> None:
         assert self._has_lock
         with open(self.lockfile, 'w', encoding='utf-8') as f:
-            f.write('PID: %s\n' % os.getpid())
+            f.write(f'PID: {os.getpid()}\n')
             if ipc_info:
                 for key in sorted(ipc_info):
-                    f.write('%s: %s\n' % (key, ipc_info[key]))
+                    f.write(f'{key}: {ipc_info[key]}\n')
 
     def release_lock(self) -> None:
         try:
             os.remove(self.lockfile)
         except OSError as e:
             if e.errno != errno.ENOENT:
                 raise
-            logger.debug('Lockfile {} not found', self.lockfile)
```

Comment:
logger should actually retain the current syntax and not use fstrings. This is a loguru (and stdlib logger as well) feature which allows the logger to decide whether or not to do the string interpolation, based on the configured log level output of the handler.

Reply:
There was a short discussion on [another PR](https://github.com/Flexget/Flexget/pull/2751#issuecomment-710082594) I made regarding the lazy evaluation of loguru and using f-strings in logs. The conclusion of this discussion was this (by @gazpachoking) :
> Given that, I'm thinking fstrings are fine. I don't think we should necessarily go replace every logging call right away, but I don't think there is a problem with it. 

I don't mind reverting it if that decision no longer stands

Reply:
Oh shit, you're right: https://loguru.readthedocs.io/en/stable/resources/migration.html#replacing-isenabledfor-method

Honestly, I kinda dislike that. @gazpachoking having the ability to lazy log on explicitly and by sending a callable, kinda sucks honestly...

Reply:
@liiight I'm not sure it really matters. We weren't using `isEnabledFor` anywhere anyway. (That I know about.) I'm fairly confident loguru does the string formatting lazily, but it isn't an advertised feature. I also don't think that really matters too much to us, given that string formatting is not super expensive, and we have debug logging enabled all the time anyway (for crash log.)